app/src/main/java/com/example/una/CharityLoginActivity.java

```diff
@@ -0,0 +1,53 @@
+package com.example.una;
+
+import androidx.appcompat.app.AppCompatActivity;
+
+import android.content.Intent;
+import android.os.Bundle;
+import android.text.Editable;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
+import android.widget.Toast;
+
+import com.example.una.models.Charity;
+import com.google.android.gms.tasks.OnFailureListener;
+import com.google.android.gms.tasks.OnSuccessListener;
+
+import org.parceler.Parcels;
+
+import butterknife.BindView;
+import butterknife.OnClick;
+
+public class CharityLoginActivity extends AppCompatActivity {
+
+    @BindView(R.id.password) EditText password;
+    @BindView(R.id.etCharityEin) EditText etCharityEin;
+    @BindView(R.id.etCharityName) EditText etCharityName;
+    @BindView(R.id.loginBtn) Button loginBtn;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_charity_login);
+    }
+
+    @OnClick(R.id.loginBtn)
+    public void logInCharity() {
+        Editable name = etCharityName.getText();
+        Editable ein = etCharityEin.getText();
+        if(name == null || name.toString() == "") {
+            Toast.makeText(this, "Please input charity name", Toast.LENGTH_SHORT).show();
```

Comment:
This isn't necessary at all but in the Survey Activity, I set the first and last name fields to be required by doing this:
```
if (firstName.isEmpty()) {
    etFirstName.setError("Required");
} else if (lastName.isEmpty()) {
    etLastName.setError("Required");
```
so if they're empty, there's a red alert that shows up in the Edit Text on the right